.prow.yaml

```diff
@@ -10,4 +10,25 @@ presubmits:
       - image: ghcr.io/kcp-dev/infra/build:1.21.8-1
         command:
         - make
-        - test
\ No newline at end of file
+        - test
+
+  - name: pull-kcp-test-e2e
```

Comment:
This should be `pull-controller-runtime-e2e` or `pull-controller-runtime-example-e2e`.